content/about/contact.rst

```diff
@@ -4,24 +4,11 @@ Contact
 :title: Contact Us
 :menu: top, Contact Us, 6; Contact Us, Contact, 1
 
-Email Addresses
-~~~~~~~~~~~~~~~
+Email
+~~~~~
 
 info@osuosl.org can be used for general inquiries about the lab.
 
-support@osuosl.org can be used for support issues with a hosted system or
-service. Note: this is for support for project infrastructure which we host; we
-cannot provide end-user support for applications that are developed or hosted
-here.
-
-abuse@osuosl.org can be used for reporting abuse to our hosted services.
-
-donations@osuosl.org can be used to discuss potential donations to the lab.
-
-press@osuosl.org can be used for press inquiries about the lab.
```

Comment:
I thought I had been told to scale down some of these emails since some of them essentially go to the same people. Maybe I misunderstood, but I can add them back on.